contentgrid-spring-common/src/main/java/com/contentgrid/spring/common/ContentGridExtensionProperties.java

```diff
@@ -0,0 +1,25 @@
+package com.contentgrid.spring.common;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import lombok.Data;
+
+@Data
+public class ContentGridExtensionProperties {
+    private Map<String, ExtensionRegistration> registration = new HashMap<>();
+
+    public Optional<ExtensionRegistration> getRegistration(String extensionName) {
+        return Optional.ofNullable(registration.get(extensionName));
+    }
+
+    @Data
+    public static class ExtensionRegistration {
+        private Map<String, URI> basePath = new HashMap<>();
```

Comment:
Can you add some examples of how different runtime platforms map to these ExtensionRegistrations?